service/scale_host_test.go

```diff
@@ -245,7 +483,8 @@ func TestWebhookCreateInvalidMinMaxActionScaleHost(t *testing.T) {
 }
 
 type MockHostDriver struct {
-	expectedConfig model.ScaleHost
+	expectedConfig1 model.ScaleHost
```

Comment:
Using "1" and "2" in the name isnt very good. Could you use something descriptive like "expecteConfigHostTemplate" and "expectedConfigLabel"?